src/Internal/Wallet/Cip30Mock.purs

```diff
@@ -210,3 +274,34 @@ mkCip30Mock pKey mSKey = do
 
 -- returns an action that removes the mock.
 foreign import injectCip30Mock :: String -> Cip30Mock -> Effect (Effect Unit)
+
+-- Helpers
+
+foreign import raisePaginateError :: forall a. Int -> Effect a
+
+foreign import _catchPaginateError
+  :: forall a. MaybeFfiHelper -> Effect a -> Effect (Maybe a)
+
+catchPaginateError :: forall a. Effect a -> Effect (Maybe a)
+catchPaginateError = _catchPaginateError maybeFfiHelper
```

Comment:
It is not used anywhere, right? If so, please remove.

Reply:
It is used in test: https://github.com/Plutonomicon/cardano-transaction-lib/pull/1264/files#diff-826108a8e51ac7cdb18bef882b2860b322b5bde19916c95705dd5ef5ef54e8a0R39

Reply:
For use in our PS API this is catched in Cip30.js

Reply:
Can we move it to the test module then?

Reply:
Then FFI will be here and we must export it. Which looks even stranger than current state to me.